chia/util/file_keyring.py

```diff
@@ -19,12 +19,14 @@
 from watchdog.observers import Observer
 
 from chia.util.default_root import DEFAULT_KEYS_ROOT_PATH
+from chia.util.errors import KeychainFingerprintNotFound, KeychainLabelExists, KeychainLabelInvalid
 from chia.util.lock import Lockfile
 
 SALT_BYTES = 16  # PBKDF2 param
 NONCE_BYTES = 12  # ChaCha20Poly1305 nonce is 12-bytes
 HASH_ITERS = 100000  # PBKDF2 param
 CHECKBYTES_VALUE = b"5f365b8292ee505b"  # Randomly generated
+MAX_LABEL_LENGTH = 30
```

Comment:
I think this is unreasonably short. At a minimum I would use a length of 64 to allow users to specify a hash or XCH address as the label (as strange as that might sound).

Reply:
See a7fc8b9710c180146d0c9fbdc0d2cf2fb0113a65